scripts/build_plugin_macos.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
```

Comment:
This doesn't look like its using anything out of `unix_share_build.sh` yet?

Reply:
no I did not touch this one yet, as I don't have access to the MACOS vm.

Reply:
So what's the next step for this file?

Reply:
maybe I can remove it from this. 
and PR it when I am back ?